vendor/bat-native-ads/src/bat/ads/internal/creatives/embeddings_database_table.cc

```diff
@@ -0,0 +1,137 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ads/internal/creatives/embeddings_database_table.h"
+
+#include <functional>
+#include <utility>
+#include <vector>
+
+#include "base/bind.h"
+#include "base/check.h"
+#include "base/strings/string_util.h"
+#include "base/strings/stringprintf.h"
+#include "bat/ads/internal/ads_client_helper.h"
+#include "bat/ads/internal/base/database/database_bind_util.h"
+#include "bat/ads/internal/base/database/database_table_util.h"
+#include "bat/ads/internal/base/database/database_transaction_util.h"
+
+namespace ads::database::table {
+
+namespace {
+
+constexpr char kTableName[] = "embeddings";
+
+std::string ConvertVectorToString(std::vector<float> vector) {
```

Comment:
This seems like it should be defined at a higher level so other functions can use it